### PR TITLE
Maintaining CamelCase

### PR DESCRIPTION
**Pull request by Maneesha-rubix**
This PR ensures that Rubix codebase maintains CamelCase for variable and function names to maintain the Go standards.

### CODE REVIEW

**Review by harirubix — APPROVED** [review]

File: block/block.go [hunk 1]
```diff
@@ -60,19 +60,19 @@ const (
...
 
 const (
-	Initiator_NLSS_share   string = "nlss_share_signature"
-	Initiator_Private_sign string = "priv_signature"
-	Initiator_DID          string = "initiator_did"
-	Initiator_Hash         string = "hash"
-	Initiator_SignType     string = "sign_type"
+	InitiatorNLSSShare   string = "nlss_share_signature"
+	InitiatorPrivateSign string = "priv_signature"
+	InitiatorDID         string = "InitiatorDID"
```
Comment (gklps) [inline block/block.go:65]: Any specific reason the string value is "InitiatorDID" and not "initiatorDID"

File: block/block.go [hunk 2]
```diff
@@ -113,11 +113,11 @@ type CreditSignature struct {
...
 
 type InitiatorSignature struct {
-	NLSS_share   string `json:"nlss_share_signature"`
-	Private_sign string `json:"priv_signature"`
-	DID          string `json:"initiator_did"`
-	Hash         string `json:"hash"`
-	SignType     int    `json:"sign_type"` //represents sign type (PkiSign == 0 or NlssSign==1)
+	NLSSShare   string `json:"nlss_share_signature"`
+	PrivateSign string `json:"priv_signature"`
+	DID         string `json:"InitiatorDID"`
```
Comment (gklps) [inline block/block.go:118]: Any specific reason the string value is "InitiatorDID" and not "initiatorDID"

**Review by gklps — COMMENTED** [review]
usage of snake case is fine with strings. Rest LGTM

File: block/block.go (hunk 1 above)
Comment (Maneesha-rubix) [reply block/block.go:65]: It's a typo. I'll modify this one in some other branch, as this PR is closed.

Comment (gklps) [reply block/block.go:65]: No worries

**Pull request merged**